src/stream/src/task/barrier_manager.rs

```diff
@@ -559,11 +567,11 @@ impl LocalBarrierWorker {
                 self.collect(actor_id, epoch)
             }
             LocalBarrierEvent::ReportCreateProgress {
-                current_epoch,
+                epoch,
```

Comment:
I suppose there's no notion of a global `current_epoch` anymore right? Hence the change. Because there could be mulitple "current_epoch" , one for each partial graph.

Reply:
I think it's not. For a single actor, a `current_epoch` only exists in partial graph. In other word, it's not possible for an actor to exist in multiple partial graphs at a same `current_epoch`.